loopy/options.py

```diff
@@ -181,6 +181,12 @@ class Options(ImmutableRecord):
         any out-of-bounds accesses.
 
         If equal to ``"no_check"``, then no check is performed.
+
+    .. attribute:: insert_additional_gbarriers
```

Comment:
```suggestion
    .. attribute:: insert_gbarriers
```
The word "additional" is confusing IMO. I know what you mean, but I think the wording is better without.

Reply:
Yeah, agreed: 7f8388aed7ad420dfa0619168bb9038b4832ebe1. (Thanks!)